polyjuice-tests/fuzz/fuzz_targets/polyjuice.rs

```diff
@@ -0,0 +1,14 @@
+#![no_main]
+#[macro_use]
+extern crate libfuzzer_sys;
+extern crate polyjuice_tests;
+use polyjuice_tests::ctx::MockChain;
+
+fuzz_target!(|data: &[u8]| {
+    let mut chain = MockChain::setup("../..").unwrap();
+    let eth_addr = [0u8; 20];
```

Comment:
We can use data[0..20] as eth_addr, [20..4] as gas_price ...., and rest as the input data.